hummingbot/connector/exchange/bittrex/bittrex_api_user_stream_data_source.py

```diff
@@ -150,6 +150,7 @@ async def listen_for_user_stream(self, ev_loop: asyncio.BaseEventLoop, output: a
                 await asyncio.sleep(30.0)
 
     async def authenticate(self):
+        self.logger().info("Authenticating...")
```

Comment:
This seems as a log introduced only for the issue investigation

Reply:
This is a message that was printed the first time this method was called, but the method is called multiple times, so I moved the message here